src/main/java/school/hei/haapi/service/StudentFileService.java

```diff
@@ -0,0 +1,44 @@
+package school.hei.haapi.service;
+
+import static school.hei.haapi.service.utils.DataFormatterUtils.formatLocalDate;
+import static school.hei.haapi.service.utils.DataFormatterUtils.instantToCommonDate;
+import static school.hei.haapi.service.utils.InstantUtils.Instant_now;
```

Comment:
rename this Instant_now to now because it is already in InstantUtils.
Avoid repeating yourself even in naming